packages/fxa-settings/Gruntfile.js

```diff
@@ -6,6 +6,7 @@
 
 module.exports = function (grunt) {
   const srcPaths = ['.license.header', 'src/**/*.ftl'];
+  const testPaths = ['../fxa-shared/l10n/branding.ftl','../fxa-react/components/**/*.ftl','src/**/*.ftl'];
```

Comment:
Adding `fxa-react` and `fxa-shared` ftl files to the test bundle to ensure that l10n tests have access to the branding terms.